### PR TITLE
sklearn is deprecated and its now called scikit-learn

### PR DESCRIPTION
**Pull request by AV25242**

### CODE REVIEW

**Comment by Court72** [comment]
@WilliamDAssafMSFT

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by WilliamDAssafMSFT** [comment]
#sign-off
